### ISSUE TITLE
Dark Theme - Activities Overview - Search text is dark on dark background.

### ISSUE THREAD

**Issue by funnylookinhat** [pop-os/gnome-shell-theme#63]
**Reporter info**

```
 * Distribution - Pop!_OS 18.04 LTS
 * Gtk+ 3.0 version - 3.22.30-1ubuntu1
 * Desktop environment - Gnome / Default
 * Pop version - 4.0.2
 ```

**Related Application and/or Shell theming (or name of widget)**
Activities Overview w/ Pop-dark-slim

**Actual issue**
When you go to search in the activities overview using the Pop Slim Dark theme, the text isn't on a useful background and cannot be seen.

**Steps to reproduce**
- Choose Pop-dark-slim theme
- Open Activities Overview
- Try to search.

**Expected behavior**
- Text should be clearly visible.

**Behavior in Adwaita**
???

**Other Note (feature-request, question, etc...)**
See screenshot:
![activities-search-bar](https://user-images.githubusercontent.com/397918/47529533-f68fa500-d85c-11e8-88ea-178be1c20432.png)

**Comment by evertiro** [comment]
Fixed with https://github.com/evertiro/gnome-shell-theme/commit/5b4601f85826066e7d72209f111df55752ceb061

![screenshot from 2018-11-05 22-08-14](https://user-images.githubusercontent.com/36433/48042511-7237fe80-e147-11e8-8a7e-41a339f49530.png)

**Comment by brs17** [comment]
This was an issue in 18.04 as well as 18.10. The issue has now been resolved in 18.10 by #71 .
